index.d.ts

```diff
@@ -280,4 +280,10 @@ declare module "replicate" {
         },
     secret: string
   ): boolean;
+
+  export function parseProgress(logs: Prediction | string): {
```

Comment:
How's the name? 

I had `parsePredictionProgress`, but that felt too wordy. Also, too specific to predictions (but maybe trainings should have a different method?)

Do you think it's worth mentioning logs in the method name?

Reply:
Yeah, I think that's clearer. Going with that.